### DISCUSSION TITLE
Hide Difficulty Adjust Mod From Scores If No Difficulty Changes Made

### DISCUSSION THREAD

**Discussion by JamieWright99** [ppy/osu#17835] [Ideas]
I use the difficulty adjust mod to set my maps to be AR 8, but when I choose to play an already AR 8 song the difficulty adjust mod will still be enabled and will show on my scores despite no changes to the difficulty. 

It would be nice if the mod hid itself from my scores when a song has no difficulty changes made, rather than me having to manually disable it and reactivate it every time I come across a map which is already AR 8.

Just an example, the attached pictures are 2 scores that are identical in difficulty, but the mod is misleading at a glance
![scores](https://user-images.githubusercontent.com/102170503/163670175-e8f32c54-0a1d-4ae8-b547-b3456bf6962f.png)

**Comment by peppy** [comment]
I think the only sane way to deal with this would be to remove the mod on entering gameplay if no changes are applied. Super low priority though.
